learn/cookbooks/strapi_v4.mdx

```diff
@@ -0,0 +1,112 @@
+
+Once you have created your account, you should be redirected to Strapi's admin dashboard. This is where you will configure your back-end API.
+
+The next step is to create a new collection type. A collection is like a blueprint of your content—in this case, it will be a collection of restaurants. You will create another collection called "Category" to organize your restaurants later.
+
+![Strapi dashboard with side menu 'Content-Type Builder' option circled](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/strapi-v4/strapi-dashboard.png)
+
+To follow along, complete "Part B:  Build your data structure with the Content-type Builder" and steps 2 to 5 in "Part D: Add content to your Strapi Cloud project with the Content Manager" from Strapi's quick start guide. These will include:
+
+- creating collection types
+- creating new entries
+- setting roles & permissions
+- publishing the content
+
+After finishing those steps of Strapi's quick start guide, two new collections named Restaurant and Category should have appeared under `Content Manager > Collection Types`. If you click on `Restaurant`, you can see that there is only one. Add more by clicking the `+ Create new entry` button in the upper-right corner of the dashboard.
+
+[Strapi dashboard: Content manager side menu, arrow indicating the location of the Restaurant Collection Type](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/strapi-v4/strapi-restaurant-content-type.png)
+
+Add the following three restaurants, one by one. For each restaurant, you need to press `Save` and then `Publish`.
+
+- Name: `The Butter Biscotte` 
+- Description: `All about butter, nothing about health.`
+
+Next, add the `French food` category on the bottom right corner of the page.
+
+[Strapi dashboard: create an entry form, arrow indicating the location of the categorie's location in the right side menu](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/strapi-v4/strapi-add-category.png)
+
+- Name: `The Slimy Snail` 
+- Description: `Gastronomy is made of garlic and butter.` 
+- Category: `French food`
+
+- Name: `The Smell of Blue` 
+- Description: `Blue Cheese is not expired, it is how you eat it. With a bit of butter and a lot of happiness.` 
+- Category: `French food`
+
+Your Strapi back-end is now up and running. Strapi automatically creates a REST API for your Restaurants collection. Check Strapi's documentation for all available [API endpoints](https://strapi.io/documentation/developer-docs/latest/developer-resources/content-api/content-api.html#api-endpoints).
+
+Now, it’s time to connect Strapi and Meilisearch and start searching through our data.
+
+## Connect Strapi and Meilisearch
+
+In order to add the Meilisearch plugin to Strapi, you need to quit our Strapi app. Go to the terminal window running Strapi and push `Ctrl+C` to kill the process.
```

Comment:
Most of the times, "in order to" can be replaced with "to" without any change to meaning.

```suggestion
To add the Meilisearch plugin to Strapi, you need to first quit the Strapi app. Go to the terminal window running Strapi and push `Ctrl+C` to kill the process.
```